Makefile

```diff
@@ -0,0 +1,36 @@
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+include $(ROOT_DIR)/.mk-lib/common.mk
+
+.PHONY: help start serve test stop restart status ps clean purge build
+
+start: ## Start all or c=<name> containers in FOREGROUND
+	@$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) up $(c)
+
+serve: ## Start all or c=<name> containers in BACKGROUND
+	@$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) up $(c) -d
+
+test: ## Execute tests
```

Comment:
Would be nice to have another option to execute an specific test or all tests from one folder as you can do it in pytest. What do you think?
I've seen that you used the django command to run the tests, but it will trigger the pytests? Can you add a parameter to add the specific test path or folder path?

Reply:
Sure, this is just an starting point :)

Tomorrow I'll explore options to inject strings or define specific string modifiers (filters).

Regarding the executed tests, it's not hardcoded, it's configurable on the invoked task so can be ammended without effort.

Currently there are no tests, pytest has not been configured and maybe it's better to deal with this in another PR